tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -869,7 +869,7 @@ def spine_ports(self, setup):
         spine_ports['alias'] = list()
 
         for key, value in minigraph_neighbors.items():
-            if (key in setup['physical_interfaces']) and ('T2' in value['name']):
+            if key in setup['physical_interfaces'] and ('T2' in value['name'] or (tbinfo['topo']['type'] == 't2' and 'T3' in value['name'])):
```

Comment:
can you split the statement, prefer 80 chars, but 120 is fine.

Reply:
@oxygen980  is this addressed?

Reply:
@rlhui yes this was done a long back.